CONTRIBUTING.md

```diff
@@ -129,6 +129,32 @@ Use the following conventions when creating a page:
+| PostgreSQL        | Postgres | A specific brand of relational database software. Always use the full name, not an informal short version. |
+| order book        | orderbook, offer book | A collection of trade orders waiting to be matched & executed, typically sorted by exchange rate. Use two words. |
+| server            | node | A server is software and/or hardware, especially the ones that connect to the XRP Ledger peer-to-peer network. The term "node" is sometimes used for this purpose but is also overloaded with other meanings including entries in a graph and Node.js, a JavaScript interpreter. |
+| stablecoin issuer | gateway | An issuer is the organization that issues a token in the XRP Ledger. A stablecoin is a token where the issuer promises that it is fully backed by some outside asset (such as fiat currency), with the stablecoin issuer providing deposit and withdraw operations to convert between the two (possibly for a fee). Previously the term "gateway" was used (especially by Ripple, the company) to describe this use case, but the rest of the industry adopted "stablecoin issuer" instead. |
+| transaction cost  | transaction fee | The amount of XRP burnt to send a transaction in the XRP Ledger. Even though this is specified in the `Fee` field of transactions, the term "fee" implies that the money is paid to someone, so "cost" is preferable. |
+| trust line        | trustline | Use two words. A trust line is a relationship between two accounts in the XRP Ledger that tracks a balance of tokens between those accounts. |
+| tokens            | IOUs, issuances, issues, issued currencies | A token in the XRP ledger may not represent money owed outside of the ledger as the name "IOU" implies. Specify "fungible tokens" if necessary to distinguish from non-fungible tokens. |
+| wallet            | wallet | Depending on the context, "wallet" could refer to hardware, software, a cryptographic key pair, or an online service. Be sure to provide enough context that the meaning is clear, or use an alternative such as "key pair" or "client application". |
+| WebSocket         | web socket, Websockets | A two way protocol for communication on the web. Always singular and in CamelCase. |
+| XRP               | XRPs, ripples | The native digital asset, or cryptocurrency, of the XRP Ledger. XRP is not a token. |
+| the XRP Ledger    | XRP Ledger (no the), Ripple, Ripple Network, RCL | The XRP Ledger was called "the Ripple network" and the "Ripple Consensus Ledger" or "RCL" at various times in the past. These names were confusing and have been retired because of their similarity to the name of the company, Ripple (formerly Ripple Labs) which develops the reference implementation of the core server. |
+| XRPL              | XRPL | Short for "XRP Ledger". As much as possible, spell out "XRP Ledger" instead; "XRPL" is cryptic and looks like it could be a typo for "XRP". |
```

Comment:
Short for _XRP Ledger_. ...spell out _XRP Ledger_ instead; _XRPL_ is cryptic and looks like it could be a typo for _XRP_.